### DISCUSSION TITLE
Angola "aoa" currency cannot be retrieved from squirephp/currencies-en

### DISCUSSION THREAD

**Discussion by clemquinones** [squirephp/squire#67] [Q&A]
I have this feature on my app that admin can set a price to certain services and have to select country then put an amount.
I use both squirephp/countries-en squirephp/currencies-en for both of country and currency selection.
Question: why is that when I select Angola, the currency "aoa" cannot be found on the currency list?

**Comment by danharrin** [comment]
Can you add it to https://github.com/squirephp/squire/blob/3.x/packages/currencies-en/resources/data.csv please?
